### ISSUE TITLE
no-scheme URL, close comment thread, export closed thread to JSON

### ISSUE THREAD

**Issue by Armonth** [posativ/isso#183]
Allow setup of a non-relative path with protocol relative URLs (1) for full support of HTTP/HTTPS without require relative-urls. On some environments relative paths are **just bad** but allowing no-scheme (protocol) URLs can fix at same time the http/https issue.

    URL_SITE = "//www.example.com"
    USE_RELATIVE_URLS = false

With this setting, we can setup a full url ( //example.com/a-awesome-post.html) and get the part of http/https of the HTML page. If a ID is needed for every thread, just add it. Then the sqlite can be moved from one site to another without problems and still don't use relative urls.

(1) http://blog.httpwatch.com/2010/02/10/using-protocol-relative-urls-to-switch-between-http-and-https/

### Close comments

Allow close threads and auto-close after X days. Don't require to explain it. Then:

### Option for export closed threads to json file

Export to a json file the comments of a closed thread. Example:

    //example.com/a-awesome-post-with-comments-closed.html
    //example.com/a-awesome-post-with-comments-closed.json

With that change, we can "drop" isso for old threads (with some code in isso or even the template if we are using something like pelican... like loading isso or json file if article is 30days older or more) and allow things like "archive"  the site (go full static site like Amazon S3).

Then, if needed (for smaller blogs is not a neccessity, for a big one... maybe?) further things can be done, for example clean that thread from sqlite to improve scalability (if we need to re-open that thread, isso can get data again from .json file).

As always, that is just a bunch of ideas :)

**Comment by posativ** [comment]
Yes, I think it would be better to have a protocol-independent uri instead of a relative URL. The migration might be trickier, but that might be a feature for 1.0 (and thus require user intervention).

Also, closed comment threads are on my todo list and I agree with all things you listed.
